src/guide/javascript.md

```diff
@@ -85,26 +85,10 @@ likely be available in the second round of this tutorial's release.
 Thus, on the plus side, configuration of the client is simple. On the
 downside, you have to prepare a lot manually.
 
-A basic client setup looks straightforward:
-
-```ts
-import { Client } from '@iroha2/client'
-
-const client = new Client({
-  torii: {
-    // Both URLs are optional - in case you need only a part of endpoints,
-    // e.g. only Telemetry ones
-    apiURL: 'http://127.0.0.1:8080',
-    telemetryURL: 'http://127.0.0.1:8081',
-  },
-})
```

Comment:
The main idea is `Client` in general requires `accountId` and `keyPair`, so this example which lacks them is misleading and that's why this has been removed, right @6r1d @0x009922 @appetrosyan ?

Reply:
To be more specific, `accountId` is always necessary, but `keyPair` is needed for performing transactions and queries.

Reply:
To be even more specific, nothing is necessary. You can pass only `torii.telemetryURL` if you only need to query for Iroha status (with `Client.getStatus()`). This case is not really relevant to the example in the tutorial. In the tutorial we need firstly get the account id and the key pair, and only then we are able to construct the client to proceed to the next chapters. I guess that is the reason why this section is removed from here and defined a bit later, after key pair generation.

Reply:
Thank you. Now I have the same understanding as you @0x009922